src/Component/Image.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Class Image
+ *
+ * @package AMP
+ */
+
+namespace Amp\AmpWP\Component;
+
+use DOMElement;
+
+/**
+ * Class Image
+ *
+ * @internal
+ * @since 1.5.0
+ */
+class Image {
```

Comment:
This class seems somewhat useless. It's just a wrapper for a `DOMElement`. Should `DOMElement` just be used instead? I suppose it is needed for extending to then have a caption.

Reply:
Hm, that's a good point. I'll look at this more.